features/cite.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <cite>
-description: The `<cite>` element TODO.
+description: The `<cite>` element is used to mark up the title of a cited creative work.
```

Comment:
I wonder if we should say that it makes the text italics by default?

Reply:
In general, I don't think we need to be consistent with MDN. As far as I know, there's never been a systematic review of those descriptions since the wiki days, so there's a good chance they're not well-considered to begin with.

I have a few style ideas here:

- [Avoiding "is used to"](https://github.com/web-platform-dx/web-features/blob/fbd5bca14baac3c2312d230d58deedd541b96139/docs/README.md#is-used-to)
- Avoiding circular descriptions (e.g., if you're not sure what "\<cite>" — the name — refers to, then our description ought to help)
- Giving an example
- Represents for consistency with the other items here

```suggestion
description: The `<cite>` element represents the title of a quoted, referenced, or mentioned creative work, such as a film or book.
```

If we want to mention default styling, we could do something like:

```suggestion
description: The `<cite>` element represents the title of a quoted, referenced, or mentioned creative work, such as a film or book. Typically styled in italic type by default.
```